src/components/modules/content-card/index.tsx

```diff
@@ -25,7 +25,11 @@ export const ContentCard = ({ title, text, button, $isNewsletter }: ContentCardP
       </StyledContentCardContent>
       {$isNewsletter && (
         <>
-          <StyledContentCardForm>
+          <StyledContentCardForm method="POST" action={action} target="_blank">
+            <div style={{ position: 'absolute', left: '-9999px' }}>
```

Comment:
There is a `.visually-hidden` class in the global styles. let's use it here, please.
```suggestion
            <div className="visually-hidden">
```